packages/react-portal-compat/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "target": "ES2019",
+    "target": "ES5",
+    "lib": ["es5", "dom"],
```

Comment:
As this package will be a dependency of v8 I used the same `lib` & `target`.

Reply:
great catch ty. 

interesting fact v8 is using es2015 features 👀 but lets keep it as is for now